.github/workflows/ci-cd-workflow.yml

```diff
@@ -88,6 +88,10 @@ jobs:
       env:
         MAGICC_EXECUTABLE_7: bin/magicc/magicc-v7.5.3/bin/magicc-darwin-amd64
       run: |
+          # Locate the gfortran library. The location can change when the image is updated
+          GFORTRAN_LIBS=$(find /usr/local/Cellar/gcc@11 -name libgfortran.5.dylib | head -n 1 | xargs dirname)
+          echo $GFORTRAN_LIBS
+          export DYLD_LIBRARY_PATH=$GFORTRAN_LIBS:$DYLD_LIBRARY_PATH
```

Comment:
@lewisjared any idea why we needed this? I've never seen this pattern before

Reply:
I'm not too sure actually. 

My working theory is that the gfortran used to build the binary was the same as the previous macOS github image so had the same paths that you had on your laptop. The changelog of the macOS image shows that there was a version bump of the gfortran@11 package last week which made it suddenly unable to locate the appropriate dylib. I'm assuming the problem is us given I have zero mac build system experience. 

The fix would be to either statically embed the gfortran lib or figure out a cross gfortran versions way of finding the appropriate dylib (there is a concept called rpath which I need to read up on) . Either way requires tweaks to the build system. But we could back port the fix for the v7 binary when resolved.

Reply:
Ah ok nice thanks for explaining!